libs/shared/utils/src/sendFormData/types.ts

```diff
@@ -26,6 +26,7 @@ export type FormAndTrackingValues = {
   phone: string;
   company: string;
   message: string;
+  url: string;
```

Comment:
for clarity perhaps, we should call this `formUrl`? In this particular context it looks like it could be the contact's URL (like their company's homepage or something).

At the very least, I think we should put a comment here explaining what `url` is.

Reply:
Thanks for keeping me honest in name choices!

Changes made; pls re-review the new version, @gabalafou.